### ISSUE TITLE
Spotboard says scoreboard has been frozen even if I did not freeze it.

### ISSUE THREAD

**Issue by hoony9x** [spotboard/domjudge-converter#3]
This happens when freeze_time and unfreeze_time are not defined in DOMjudge.
In this case, "noMoreUpdate" in "runs.json" sets to 'null' and spotboard says "scoreboard has been frozen".;

**Comment by myungwoo** [comment]
Below [line](/spotboard/domjudge-converter/blob/ab7ecd70125665011b653567ffab10aefe251d44/index.js#L188) was a problem:
```javascript
this.frozen = (contest.freeze && now >= contest.freeze) && !(config.unfreeze || contest.unfreeze && now >= contest.unfreeze);
```

It's because of Javascript's [Logical Operators](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_Operators) doesn't always return a boolean value.

**Comment by myungwoo** [comment]
Solved by 0b0cd036951e7c47635bacb147c04875c3f70c65
